sycl/include/sycl/detail/vector_convert.hpp

```diff
@@ -58,7 +58,6 @@
 #include <sycl/exception.hpp>                  // for errc
 
 #include <sycl/ext/oneapi/bfloat16.hpp> // bfloat16
-#include <sycl/vector.hpp>
```

Comment:
TODO: revert this back. `vector.hpp` should not include `vector_convert.hpp`, because the latter is heavy and it is not used everywhere. We should separately include both from `sycl.hpp` and update all tests/library files which require converts to be present to include them explicitly

Reply:
@aelovikov-intel, I've reverted those changes in b38ceed5718bbecb7ef866cba66e4651f6ef560f, the patch should be ready for your review